packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -56,18 +53,10 @@ export default function useTabNav() {
 		}
 	}
 
-	const before = (
-		<div
-			ref={ focusCaptureBeforeRef }
-			tabIndex={ focusCaptureTabIndex }
-			onFocus={ onFocusCapture }
-		/>
-	);
-
 	const after = (
 		<div
 			ref={ focusCaptureAfterRef }
-			tabIndex={ focusCaptureTabIndex }
+			tabIndex={ ! isNavigationMode ? 0 : undefined }
```

Comment:
If in navigation mode, this tab stop goes away. If not, this prevents us from entering in to another block vs. going to the sidebar.